extensions/amp-next-page/amp-next-page.md

```diff
@@ -25,237 +25,272 @@ limitations under the License.
-| `imageUrl` (required) | String URL of the image to display in the recommendation box                                                                                                     |
+### `deep-parsing` (optional)
 
-### Example configuration
+When specified, the `deep-parsing` attribute enables recursive parsing of inline JSON configurations from subsequently loaded documents. This only works if the loaded documents contain an `<amp-next-page>` tag.
 
-The following configuration will recommend two more documents for the user to
-read, and hides the header and footer elements from each child document.
+This behavior is the default if the `<amp-next-page>` component only specifies an inline configuration. If the `<amp-next-page>` component loads the configuration remotely this behavior is disabled by default unless `deep-parsing` is set to `true`.
 
-```json
-{
-  "pages": [
-    {
-      "image": "https://example.com/image1.jpg",
-      "title": "This article shows first",
-      "ampUrl": "https://example.com/article1.amp.html"
-    },
-    {
-      "image": "https://example.com/image2.jpg",
-      "title": "This article shows second",
-      "ampUrl": "https://example.com/article2.amp.html"
-    }
-  ],
-  "hideSelectors": [".header", ".footer"]
-}
-```
+### `xssi-prefix` (optional)
 
-## Substitutions
+When specified, this attribute enables `amp-next-page` to strip a prefix before parsing remotely hosted JSON. This can be useful for APIs that include [security prefixes](http://patorjk.com/blog/2013/02/05/crafty-tricks-for-avoiding-xssi/) like `)]}` to help prevent cross site scripting attacks.
 
-The `amp-next-page` src allows all standard URL variable substitutions. See
-the [Substitutions Guide](../../spec/amp-var-substitutions.md) for more info.
+## Styling
 
-For example:
+The `<amp-next-page>` component renders a default recommendation box and separator. You may style the appearance of these two elements as follows:
+
+### Styling the default footer recommendation box
+
+The default footer recommendation box (that appears when no more suggestions are available or the suggestions exceed the maximum specified number of pages) can be styled by appling CSS styles to the following classes:
+
+- `.amp-next-page-footer` for the parent container element
+- `.amp-next-page-footer-content` for the wrapping element
+- `.amp-next-page-footer-article` for an individual article which contains:
+- `.amp-next-page-footer-image` for the link image
+- `.amp-next-page-footer-text` for the link text
+
+### Styling the default separator
+
+The default separator (shown between article recommendations) is a simple gray hairline, although this can be customized through CSS by applying styles to the `.amp-next-page-separator` class, templating the separator allows for more flexibility in styling and allows the integration of the title, image and URL of the upcoming article into the separator's appearance.
+
```

Comment:
remove this.

Reply:
I think this is needed, again to differentiate between the default and custom implementations of a separator